appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -16,31 +16,46 @@ type ExtendedSkuDescription = WebSiteManagementModels.SkuDescription & { label?:
 
 export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
     public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
-        let skus: ExtendedSkuDescription[] = wizardContext.advancedCreation ? this.getRecommendedSkus().concat(this.getAdvancedSkus()) : this.getRecommendedSkus();
-        if (wizardContext.newSiteKind === AppKind.functionapp) {
-            skus.push(...this.getElasticPremiumSkus());
-        } else if (wizardContext.newSiteKind?.includes(AppKind.workflowapp)) {
-            skus = this.getWorkflowStandardSkus();
-        }
-
-        const regExp: RegExp | undefined = wizardContext.planSkuFamilyFilter;
-        if (regExp) {
-            skus = skus.filter(s => !s.family || regExp.test(s.family));
-        }
-
-        const pricingTiers: IAzureQuickPickItem<WebSiteManagementModels.SkuDescription | undefined>[] = skus.map(s => {
+        const pricingTiers: IAzureQuickPickItem<WebSiteManagementModels.SkuDescription | undefined>[] = this.getRecommendedSkus().map(s => {
             return {
                 label: s.label || nonNullProp(s, 'name'),
                 description: s.description || s.tier,
-                data: s
+                data: s,
+                group: wizardContext.advancedCreation ? 'Recommended' : ''
             };
         });
 
+        let advancedSkus: ExtendedSkuDescription[] = [];
+        if (wizardContext.advancedCreation) {
+            advancedSkus.push(...this.getAdvancedSkus());
+
+            if (wizardContext.newSiteKind === AppKind.functionapp) {
+                advancedSkus.push(...this.getElasticPremiumSkus());
+            } else if (wizardContext.newSiteKind?.includes(AppKind.workflowapp)) {
+                advancedSkus = this.getWorkflowStandardSkus();
+            }
+
+            const regExp: RegExp | undefined = wizardContext.planSkuFamilyFilter;
+            if (regExp) {
+                advancedSkus = advancedSkus.filter(s => !s.family || regExp.test(s.family));
+            }
+
+            pricingTiers.push(...advancedSkus.map(s => {
+                return {
+                    label: s.label || nonNullProp(s, 'name'),
+                    description: s.description || s.tier,
+                    data: s,
+                    group: 'Additional Options',
+                };
+            }));
+        }
+
         pricingTiers.push({ label: localize('ShowPricingCalculator', '$(link-external) Show pricing information...'), data: undefined, suppressPersistence: true });
```

Comment:
You should update the dependency of the ui package in package.json because we don't want anyone to use this unless they have your corresponding fix there.